crates/ibc/src/core/context.rs

```diff
@@ -612,10 +719,13 @@ mod val_exec_ctx {
         )?;
 
         let conn_id_on_b = msg.connection_hops_on_b[0].clone();
-        let port_channel_id_on_b = (msg.port_id_on_b.clone(), chan_id_on_b.clone());
 
         // emit events and logs
```

Comment:
Would the logged events be removed if something fails in the next `state change` step? is this a possible case?

Reply:
Good point [d65553c](https://github.com/cosmos/ibc-rs/pull/352/commits/d65553c923e4d30b54796c8e2168751bf75111b4)